rpc_jobs/rpc_aio.yml

```diff
@@ -47,8 +54,6 @@
             Cleanup,
             Destroy Slave
           UPGRADE_FROM_REF: "liberty-12.2"
```

Comment:
If we need to cater for more major upgrades then moving this into series seems to make sense.